pyproject.toml

```diff
@@ -1,27 +1,28 @@
 [tool.poetry]
 name = "panorama"
-version = "0.2.2"
+version = "0.2.3"
 description = ""
 authors = ["Thomas Brockmeier <t.brockmeier@amsterdam.nl>"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-httpx = "^0.23.0"
-pydantic = "^1.8.2"
+python = "~3.9"
+httpx = {version = "^0.23.0", source = "PyPI"}
```

Comment:
This makes poetry way faster in running `poetry update` in case you have multiple sources.
We don't have this case, but it could be that a project that uses this package has multiple sources, and then this speeds it up.